### ISSUE TITLE
ember-cli-babel 5.x has been deprecated

### ISSUE THREAD

**Issue by petey** [knownasilya/ember-highlight#11]
```
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: ember-app -> ember-highlight -> ember-cli-babel
```

**Comment by knownasilya** [comment]
v1.2.6 should already be 6.6, give that a try.

**Comment by knownasilya** [comment]
I released v1.2.7 which has updated lock files, which was probably the issue.
